.github/workflows/deploy-clock-app.yml

```diff
@@ -27,23 +27,9 @@ jobs:
             # yamllint disable-line rule:line-length
           cli_commands: edge-app upload --app-id=${{ env.APP_ID }} --path=${{ env.APP_PATH }}
 
-      - name: Get Revisions
-        id: get_revisions
-        uses: screenly/cli@master
-        with:
-          screenly_api_token: ${{ secrets.SCREENLY_API_TOKEN }}
-            # yamllint disable-line rule:line-length
-          cli_commands: edge-app version list --app-id=${{ env.APP_ID }} --path=${{ env.APP_PATH }} --json
-
-      - name: Latest Revision
-        # Using `jq` to extract the latest revision
-        id: latest_revision
-          # yamllint disable-line rule:line-length
-        run: echo "revision=$(echo ${{ steps.get_revisions.stdout.value }} | jq '.[-1:] | .[].revision')" >> "$GITHUB_OUTPUT"
```

Comment:
This line fails as well. To make it work need to change `{{ steps.get_revisions.stdout.value }` to `{{ steps.get_revisions.outputs.cli_commands_response }` with my changes from https://github.com/Screenly/cli/pull/144